internal/runners/projects/projects.go

```diff
@@ -21,9 +26,12 @@ type projectWithOrg struct {
 	Organization   string   `json:"organization"`
 	LocalCheckouts []string `json:"local_checkouts,omitempty"`
 	Executables    []string `json:"executables,omitempty"`
+	LastUsed       []string `json:"last_used,omitempty"`
 }
 
-func newProjectWithOrg(name, org string, checkouts []string) projectWithOrg {
+const runtimeInUseCutoff = 90 * time.Second // heartbeats are usually sent once per minute
```

Comment:
Please have this share a central value with the watcher, ie.

https://github.com/ActiveState/cli/blob/79cb7134123e7836b1c9fd52111dd274aea956e4/cmd/state-svc/internal/rtwatcher/watcher.go#L21-L21

We can probably just move that to constants.